pkg/generate/cleanup/controller.go

```diff
@@ -188,7 +188,7 @@ func (c *Controller) deleteGR(obj interface{}) {
 			return
 		}
 
-		if r != nil && r.GetLabels()["policy.kyverno.io/synchronize"] == "enable" {
+		if r != nil && r.GetLabels()["generate.kyverno.io/synchronize"] == "enable" {
```

Comment:
Will changing the label impact compatibility with existing GRs after an upgrade?

Reply:
Yes, changing the label will impact compatibility with existing GRs after an upgrade